dashboard/app/controllers/levels_controller.rb

```diff
@@ -236,8 +232,10 @@ def level_params
       :level_num,
       :user,
       :dsl_text,
+      :encrypted,
       {concept_ids: []},
-      {soft_buttons: []}
+      {soft_buttons: []},
+      {examples: []}
```

Comment:
is this `examples` parameter important related to this PR somehow, or leftover/typo from something else?